Formula/crfsuite.rb

```diff
@@ -0,0 +1,41 @@
+class Crfsuite < Formula
+  desc "Fast implementation of conditional random fields"
+  homepage "https://www.chokkan.org/software/crfsuite/"
+  url "https://github.com/downloads/chokkan/crfsuite/crfsuite-0.12.tar.gz"
+  sha256 "e7fc2d88353b1f4de799245f777d90f3c89a9d9744ba9fbde8c7553fa78a1ea1"
+  license "BSD-3-Clause"
+
+  depends_on "python@3.11" => :test
```

Comment:
Do we build python bindings for it?
Should we add `:build` for `python@3.11` as well? Or maybe switching to `uses_from_macos "python"`?

Reply:
Switching to `uses_from_macos "python"`. This package does not build Python bindings. An example is provided in the repo which uses a Python script to convert the data set resource that we download into a `crfsuite`-compatible format. This Python script is not really considered as part of the application (or, at least `make install` doesn't install it).